assets/js/components/data/index.js

```diff
@@ -77,14 +77,10 @@ const dataAPI = {
 	maxRequests: 10,
 
 	init() {
-		global.googlesitekit.initialized = true;
-		this.collectModuleData = this.collectModuleData.bind( this );
-		global.googlesitekit.cache = [];
-
 		addAction(
 			'googlesitekit.moduleLoaded',
 			'googlesitekit.collectModuleListingData',
-			this.collectModuleData
+			this.collectModuleData.bind( this )
```

Comment:
@aaemnnosttv Not sure about this change, as it no longer reassigns the value like above. Unless there's a good reason not to, I'd prefer we stuck with that.

Reply:
I inlined this as it seems to be the only place it's ever used. The use of `this` in `dataAPI` is a bit odd as it's never instantiated. I'm not sure there's any benefit to exposing this method. I'm open to keeping the old assignment though if you'd like.

Reply:
Let's keep it as you have it now.